sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,7 @@
+SELECT name AS genre,
+       COUNT("filmGenre"."genreId") AS total
+  FROM genres
+  JOIN "filmGenre" USING ("genreId")
+  JOIN "castMembers" USING ("filmId")
+ WHERE "actorId" = 178
+ GROUP BY name
```

Comment:
Your query should find the right actor, regardless of which id they have:

---

![image](https://user-images.githubusercontent.com/16143771/180912458-a91e05f1-6fc3-433c-b805-6f4fe09c02ce.png)